src/bindings/python/tests/mock/ov_mock_py_frontend/mock_py_frontend.hpp

```diff
@@ -603,26 +611,27 @@ struct MOCK_API FeStat {
     int get_name() const {
         return m_get_name;
     }
+
     int supported() const {
         return m_supported;
     }
 };
 
-class MOCK_API FrontEndMockPy : public FrontEnd {
-    mutable FeStat m_stat;
+class MOCK_API FrontEndMockPy : public IFrontEnd {
+    static FeStat m_stat;
```

Comment:
Why do we need global member?

Reply:
This is because previously we were able to cast 'FrontEnd' to 'FrontEndMockPy' in test python code. Now 'FrontEndMockPy' class is not accessible from 'FrontEnd' (we didn't agree to add `IFrontEnd& FrontEnd::getImpl() {return _actual;}`).
For Python test it is enough to verify that 'function was called', we don't have complicated tests that 'function was called for particular frontend instance'. So making statistics as 'static' looks good approach. But it also required to introduce 'clear_statistics' methods before running each test case